app/src/views/private/components/drawer-item.vue

```diff
@@ -403,6 +403,10 @@ function useActions() {
 			validationErrors.value = [];
 		}
 
+		if (props.junctionField && Object.values(defaultValues.value).find((value) => value !== null)) {
```

Comment:
```suggestion
		if (props.junctionField && Object.values(defaultValues.value).some((value) => value !== null)) {
```

I believe we need to use `.some` here because `.find` is returning the value.

Especially when the default value is `0` or `false`, it will then become `if (props.junctionField && 0)` or `if (props.junctionField && false)`, which are both still falsy. Please do let me know if `.some` is still not accurate 👌